src/emc/sai/saicanon.cc

```diff
@@ -1001,40 +889,40 @@ extern int GET_EXTERNAL_QUEUE_EMPTY()
 /* Returns the system value for spindle speed in rpm */
 double GET_EXTERNAL_SPEED(int spindle)
 {
-  return _spindle_speed[spindle];
+  return _spindle_speed;
 }
```

Comment:
Same here as above - does this remove multi-spindle support??

Reply:
This is a mistake, good catch! Anything like this that steps on multispindle is a mistake, due to bad resolution of rebase conflicts. Multispindle should not be affected by my changes, so I'll go through and fix these up.